ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Thanks for using Certbot - if you're experiencing issues with using Certbot
+we'd appreciate it if you could answer a few questions about your environment so
+that we can better assist you:
+
+## My operating system is (include version):
+
+
+## My web server is (include version):
+
+
+## How did you install Certbot:
+
+
+## What command did you run? What output did it produce?
+### Please provide the Certbot error log if you can. It is stored by default in `/var/log/letsencrypt`
```

Comment:
I'd make this it's own question like "Can you provide a Certbot log showing the issue?".

If you agree with me about not asking for the domain, I think we should also include wording like the following somewhere:

> Feel free to redact domain names and e-mail and IP addresses as you feel appropriate.

We should also include the sentence about where logs are stored.

I'm not sure of the best way to format this though. If you look at the rendered markdown, the domain question looks really weird with a small paragraph in bold text. I guess this line looks better in slightly softer text under the main question, but still looks a bit off to me. If you can find something you're happy with though then that works for me.